config/build.sh

```diff
@@ -47,6 +54,12 @@ yum groupremove -y "Development Tools" && \
 yum -y remove ${GIT_DEPENDENCIES}
 yum clean all
 yum -y autoremove
+
+# autoremove removes ssh (which it presumably wouldn't if we were able
+# to install git from a repository, because git has a dep on ssh.)
+# Do we care to restrict this to a particular version?
+yum -y install openssh-clients
```

Comment:
Let's create a new variable called something like RUNTIME_DEPENDENCES and add `curl`, `openssh-clients` and other packages removed by the call that removes all the `*-devel` packages.

Reply:
ditto.